src/sink/wavefront.rs

```diff
@@ -335,6 +348,7 @@ mod test {
         assert!(lines.contains(&"test.timer.99 12.101 645181811 source=test-src"));
         assert!(lines.contains(&"test.timer.999 12.101 645181811 source=test-src"));
         assert!(lines.contains(&"test.timer.count 3 645181811 source=test-src"));
+        assert!(lines.contains(&"test.timer.mean 5.434333333333334 645181811 source=test-src"));
```

Comment:
Oh wait, maybe I'm misunderstanding your comments. The CKMS type in cernan is specialized to `CKMS<f64>` and everything is coded to that demand. It's only now that we've updated to quantiles 0.4.0 that `CKMS<T>` must be `T: Into<f64>`. The impact on cernan is nil.